hosts

```diff
@@ -264,7 +272,7 @@ ioc_si_ps_trims_m12_srv_host=lnls449-linux
 ioc_si_ps_diags_srv_host=lnls452-linux
 
 [ioc_si_ps_quad_q_srv]
-lnls558-linux
+lnls561-linux
```

Comment:
wouldn't it be lnls451-linux?

Reply:
or maybe we do not need this separation anymore

Reply:
we dont need this host group anymore. deleted!